.github/zizmor.yml

```diff
@@ -9,13 +9,18 @@ rules:
   cache-poisoning:
     ignore:
       - build-docker.yml
-      - publish-playground.yml
-      - ty-ecosystem-analyzer.yaml
-      - ty-ecosystem-report.yaml
   excessive-permissions:
     # it's hard to test what the impact of removing these ignores would be
     # without actually running the release workflow...
     ignore:
       - build-docker.yml
-      - publish-playground.yml
       - publish-docs.yml
+  secrets-inherit:
+    # `cargo dist` makes extensive use of `secrets: inherit`,
+    # and we can't easily fix that until an upstream release changes that.
+    disable: true
```

Comment:
I went ahead and blanket-disabled these findings, since we don't have a remediation strategy until `cargo dist` uses fine-grained secret inheritance.